README.md

```diff
@@ -57,6 +57,8 @@ Developer Setup
       - Once installed, right click the eclipse launchers found in 'triplea/eclipse/launchers'
       - Select 'convert with eclipser', then the launchers will be available to run as intelliJ run configurations
 
+Note that eclipse and NetBeans are currently not supporting JUnit 5. Therefore we use the JUnit 4 based Runner `JUnitPlatform.class` until there is official support.
```

Comment:
I think we could use a bit more here - explicitly answer what exactly someone needs to do with eclipse to run the tests. It sounds like that  with the referred runner, that junit5 test classes can be run as normal junit4. Is that the case? More detail on the JunitPlatform.class could be of benefit as well, in a few words, where and how is it used?